src/wfs/src/test/java/org/geoserver/wfs/v1_1/VersionNegotiationTest.java

```diff
@@ -45,7 +45,10 @@ public void test0() throws Exception {
         request.getAcceptVersions().getVersion().add("1.1.0");
 
         TransformerBase tx = getCaps.run(GetCapabilitiesRequest.adapt(request));
-        assertTrue(tx instanceof CapabilitiesTransformer.WFS1_1);
+        assertTrue(
+                tx
+                        instanceof
+                        CapabilitiesTransformer.WFS1_0); // choose first appropriate AcceptVersions
```

Comment:
The formatter mandates 100 columns, best not put trailing comments unless very short

```suggestion
        // choose first appropriate AcceptVersions
        assertTrue(tx instanceof CapabilitiesTransformer.WFS1_0); 
```